_meta/fields.common.yml

```diff
@@ -82,6 +134,60 @@
           type: group
           fields:
 
+          - name: body
+            type: object
+            enabled: false
+            indexed: false
+            analyzed: false
+            searchable: false
+            aggregatable: false
+
+          - name: cookies
+            type: object
+            enabled: false
+            indexed: false
+            analyzed: false
+            searchable: false
+            aggregatable: false
+
+          - name: headers
+            type: object
+            enabled: false
+            indexed: false
+            analyzed: false
+            searchable: false
+            aggregatable: false
+            # intake enforces:
+            # fields:
+            # - name: content-type
+            #   type: keyword
+            # - name: cookie
+            #   type: keyword
+            # - name: user-agent
+            #   type: keyword
+
+          - name: env
+            type: object
+            enabled: false
+            indexed: false
+            analyzed: false
+            searchable: false
+            aggregatable: false
+
+          - name: socket
+            type: object
+            enabled: false
+            indexed: false
+            analyzed: false
+            searchable: false
+            aggregatable: false
+            # intake enforces:
+            # fields:
+            # - name: socket.encrypted
+            #   type: boolean
+            # - name: socket.remote_address
+            #   type: keyword
```

Comment:
Is it possible to also define the nested fields here?

Reply:
I want to so they'll show up in documentation but not as-is.  Did you have other concerns besides docs?

Reply:
No concerns, I was hoping we could add them.